analysers/analyser_merge_tourism_offices_FR.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Noémie Lehuby 2020                                         ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Datatourisme_tourism_office_FR(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item = 8010, id = 310, level = 3, tags = ['merge'],
+            title = T_('Tourism office not integrated'))
+
+        self.init(
+            "https://data.datatourisme.gouv.fr",
+            "DATAtourisme, la base nationale des données du tourisme en Open Data",
+            CSV(Source(attribution = "data.gouv.fr:DATAtourisme", millesime = "05/2020",
+                    fileUrl = "https://diffuseur.datatourisme.gouv.fr/webservice/9e8b7142a9fe83b82225032611cdb57e/cb33fad9-e86e-4f8a-a105-f4472f720526")),
+            Load("Longitude", "Latitude",
+                uniq = ["elem"]),
+            Mapping(
+                select = Select(
+                    types = ["nodes", "ways"],
+                    tags = {"information": "office"}),
+                conflationDistance = 300,
+                generate = Generate(
+                    static1 = {
+                        "information": "office",
+                        "tourism": "information"},
+                    static2 = {"source": self.source},
+                    mapping1 = {
+                        "contact:phone": "contact_phone",
```

Comment:
```suggestion
                        "wheelchair": lambda res: "yes" if res["reducedMobilityAccess"] == "true" else None,
                        "contact:phone": "contact_phone",
```

Reply:
Too late, Noemie already did this.

Reply:
ok, and i think it's better, but i don't understand how you link wheelchair, parse_wheelchair and column "reducedMobilityAccess"

Reply:
You are right, the column name is `reducedMobilityAccess` not, `wheelchair`.
You are good to a new PR :].

Reply:
sorry, i try to change https://github.com/osm-fr/osmose-backend/pull/865/files put i can't ? because it's merged i can't modify ?

Reply:
Need to restart from the master to make a new PR https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_tourism_offices_FR.py

Reply:
https://github.com/osm-fr/osmose-backend/pull/872

Reply:
I've updated the requests earlier this day, to keep consistent column headers. The field will be called "wheelchair" when it will be generated again (tomorrow morning)